### PR TITLE
Interactive buttons

### PR DESCRIPTION
**Pull request by xnkevinnguyen**
# Description:
Modify microbit svg to make it easier for the user to interact with buttons.

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Limitations:

Please describe limitations of this PR

# Testing:

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

# Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] My code has been formatted with `npm run format` and passes the checks in `npm run check`
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/view/styles/Microbit.css [hunk 1]
```diff
@@ -14,16 +17,24 @@ svg.sim.grayscale {
...
 }
 .sim-board,
 .sim-display,
 sim-button {
     fill: #111;
 }
 .sim-button-outer:hover {
-    stroke: grey;
-    stroke-width: 3px;
+    stroke: orange !important;
```
Comment (nasadigital) [inline src/view/styles/Microbit.css]: Do you think we can avoid using `!important`?

File: src/view/styles/Microbit.css [hunk 2]
```diff
@@ -14,16 +17,24 @@ svg.sim.grayscale {
 }
 .sim-button {
     pointer-events: none;
+    fill: "rgb(17, 17, 17)";
+}
+.sim-button:active {
```
Comment (nasadigital) [inline src/view/styles/Microbit.css:22]: Are we going to give the metal strips at the bottom the same treatment (or are they not meant to be interacted with in the same way)?

Comment (xnkevinnguyen) [reply src/view/styles/Microbit.css:22]: The metal strips currently don't have functionality. We would add the same treatment if we implement functionality on those in the future.

File: src/view/styles/Microbit.css (hunk 1 above)
Comment (xnkevinnguyen) [reply src/view/styles/Microbit.css]: Yes! Removing it. Shouldn't be used when not necessary. Good catch

**Review by nasadigital — APPROVED** [review]
LGTM!

**Pull request merged**